README.md

```diff
@@ -27,15 +27,15 @@ You want to contribute to go-clang? GREAT! If you are here because of a bug you
 
 This repository, [gen](https://github.com/go-clang/gen), holds the code to generate new bindings from headers of Clang's C API. These bindings are then bootstrapped using the [bootstrap](https://github.com/go-clang/bootstrap) repository. The `bootstrap` repository holds all basic files, like the CI configuration and a Makefile, as well as some additional code to make the bindings more complete and powerful.
 
-To ease the development process we have our own development environment based on [Vagrant](https://www.vagrantup.com/). The provided Vagrantfile executed in the root of the repository will setup an Ubuntu VM with our currently used Go version as well as Clang 3.4 and will set up everything that is needed to development and handle new versions of Clang.
+To ease the development process we have our own development environment based on [Docker](https://www.docker.com/). The provided `go-clang/build` `docker` image (located under `docks/build`) executed in the root of the repository will setup an Ubuntu image with our currently used Go version as well as Clang 3.4 and will set up everything that is needed to development and handle new versions of Clang.
```

Comment:
Finally I have a big enough block to review this :-)
- I am all for eliminating Vagrant for Docker. The development environment is simple enough to fully rely on docker.
- If we are talking about removing `vagrant-bootstrap.sh` and `vagrant-environment.sh` that is fine with me.